### PR TITLE
Add rdf:dirLangString to discussion on values spaces

### LINKED ISSUE #120: Update the note about the lexical space of rdf:langString to include rdf:dirLangString
The note https://www.w3.org/TR/rdf12-concepts/#h-note-9 (current link - 2024-12-12) should also include rdf:dirLangString .

### PR DESCRIPTION
**Pull request by afs**
This closes #120

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/pull/121.html" title="Last updated on Dec 12, 2024, 9:48 PM UTC (0b53c4c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/121/1f0b49f...0b53c4c.html" title="Last updated on Dec 12, 2024, 9:48 PM UTC (0b53c4c)">Diff</a>

### CODE REVIEW

**Review by Tpt — APPROVED** [review]

**Review by gkellogg — APPROVED** [review]

File: spec/index.html [hunk 1]
```diff
@@ -1115,14 +1115,20 @@ <h2>Datatypes</h2>
...
     of <a>datatypes</a> does not accommodate
     <a>language tags</a> in the <a>lexical space</a>.
     The <a>value space</a> associated with this datatype IRI is the set
-    of all pairs of strings and language tags.</p>
+    of all pairs of strings and language tags.
+    Similarly, <a>directional language-tagged strings</a> have the <a>datatype IRI</a>
+    <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#dirLangString</code>
+    (commonly abbreviated as <code>rdf:dirLangString</code>).
+    The <a>value space</a> associated with this datatype IRI is the set
+    of all 3-tuples of strings, language tags and directions.
```
Comment (hartig) [inline spec/index.html:1130]:
```suggestion
    of all 3-tuples that consist of a string, a language tag,
    and a <a>base direction</a>.
```

File: spec/index.html [hunk 2]
```diff
@@ -1115,14 +1115,20 @@ <h2>Datatypes</h2>
...
-  <p class="note"><a>Language-tagged strings</a> have the <a>datatype IRI</a>
+  <p class="note" id="note-lang-strings"><a>Language-tagged strings</a> have the <a>datatype IRI</a>
     <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>
     (commonly abbreviated as <code>rdf:langString</code>).
     No datatype is formally defined for this IRI because the definition
     of <a>datatypes</a> does not accommodate
     <a>language tags</a> in the <a>lexical space</a>.
     The <a>value space</a> associated with this datatype IRI is the set
-    of all pairs of strings and language tags.</p>
+    of all pairs of strings and language tags.
```
Comment (hartig) [inline spec/index.html:1125]:
```suggestion
    of all pairs that consist of a string and a language tag.
```

**Review by hartig — CHANGES_REQUESTED** [review]

File: spec/index.html (hunk 1 above)
Comment (afs) [reply spec/index.html:1130]: Happy to - I copied the style in the existing text.

Comment (afs) [reply spec/index.html]: Done, with consistent introduction of links.

File: spec/index.html (hunk 2 above)
Comment (afs) [reply spec/index.html]: Done.

**Review by hartig — APPROVED** [review]

File: spec/index.html [hunk 3]
```diff
@@ -1115,14 +1115,21 @@ <h2>Datatypes</h2>
...
     <a>language tags</a> in the <a>lexical space</a>.
     The <a>value space</a> associated with this datatype IRI is the set
-    of all pairs of strings and language tags.</p>
+    of all pairs that consist of a string and a language tag.
+    Similarly, <a>directional language-tagged strings</a>
+    <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#dirLangString</code>
+    (commonly abbreviated as <code>rdf:dirLangString</code>)
+    also have a <a>base direction</a> in the value space.
+    The <a>value space</a> associated with this datatype IRI is the set
+    of all 3-tuples of a string, a language tag and a base direction.
```
Comment (TallTed) [inline spec/index.html:1131]:
```suggestion
    of all 3-tuples of a string, a language tag, and a base direction.
```

**Comment by domel** [comment]
See PR https://github.com/w3c/rdf-schema/pull/31

**Pull request merged**